teuthology/task/internal/__init__.py

```diff
@@ -334,6 +339,20 @@ def fetch_binaries_for_coredumps(path, remote):
             remote.get_file(debug_path, coredump_path)
 
 
+def gzip_if_too_large(min_size_config, src, tarinfo, local_path):
+    try:
+        min_size = humanfriendly.parse_size(min_size_config)
+    except humanfriendly.InvalidSize:
+        msg = 'invalid "log_compress_min_size": {}'.format(min_size_config)
+        log.error(msg)
+        raise ConfigError(msg)
```

Comment:
1) The code of parsing the size is better to place close to reading the value from the config.
2) Also we need to update the doc describing the `log_compress_min_size` parameter usage and value format.